frontend/src/stateless/index.ts

```diff
@@ -317,44 +317,29 @@ function generateSecureToken(length = 16): string {
  * @param isStateless
  * @returns true if the present stored config is different from the fetched one.
  */
-export function processClusterComparison(
-  clusters: ConfigState['clusters'],
-  clustersToConfig: ConfigState['clusters'],
-  isStateless: boolean
-) {
-  if (!clusters) {
-    // If no clusters, then add all clusters
-    return false;
+export function isClusterConfigDifferent(
```

Comment:
For boolean values it is usually better to name them positive rather than negative, like `isEnabled` rather than `isDisabled`, it makes it easier to reason later on, for example `if(isEnabled)` is easier to read than `if(!isDisabled)`. 

So in this case I'd sugget naming it `isEqualClusterConfigs`